app/scripts/views/home/home.js

```diff
@@ -3,19 +3,75 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 define([
+  'jquery',
+  'fullpage',
   'views/base'
-], function (BaseView) {
+], function ($, fullpage, BaseView) {
   'use strict';
 
   var HomeView = BaseView.extend({
+    events: {
+      'click .skip-intro':'_skipIntro',
+      'click .arrow': '_moveDown'
```

Comment:
aha, so it does handle scroll and up/down arrows, although I think it might be a little bit too slow of an animation, it's maddening if you just want to skip the whole thing by scrolling. can we shorten the animation time? thoughts?